apps/docs/content/guides/platform/manage-your-usage/disk-iops.mdx

```diff
@@ -0,0 +1,71 @@
+---
+id: 'manage-usage-disk-throughput'
+title: 'Manage Disk IOPS usage'
+---
+
+## What you are charged for
+Each database has a dedicated disk. You are charged for the provisioned disk IOPS. You can configure disk IOPS in the advanced settings on the [project's Compute and Disk page](https://supabase.com/dashboard/project/_/settings/compute-and-disk).
```

Comment:
Maybe add a sentence that there are no additional IOPS charges unless specifically opting into extra provisioned IOPS